src/main/java/seedu/address/logic/parser/FindCommandParser.java

```diff
@@ -28,7 +28,7 @@ public FindCommand parse(String args) throws ParseException {
         String[] nameKeywords = trimmedArgs.split("\\s+");
 
         // TODO: Implement the logic for searching by NRIC
```

Comment:
Could you remove this comment since it is completed in your feature

Reply:
Okk